shared/localization/swap-locale.js

```diff
@@ -5,7 +5,9 @@
  */
 'use strict';
 
-const {set: _set, get: _get} = require('lodash');
+// Using these rather than the full 'lodash' saves 134kb to the report/bundle.umd.js
+const _get = require('lodash.get');
+const _set = require('lodash.set');
```

Comment:
Did it really?

When I reviewed #13695 I checked for this and confirmed that all the extra lodash stuff was getting removed from the bundle.

Reply:
> Did it really?

yup.

![image](https://user-images.githubusercontent.com/39191/165345838-30185899-72a8-4cec-b1ff-b8b76823c24d.png)

I also tried moving the two localization/*.js files to esm so the lodash import was esm for treeshaking and i didnt get any savings in that case either.

I did not know about this CVE thing though.

Reply:
> I did not know about this CVE thing though.

also annoying.. the [alternatives](https://github.com/lukeed/dset#related) to lodash's get/set i'm finding don't use the square bracket notation we've been using... eg `'audits[is-on-https].title'`. (They just use `'audits.is-on-https.title'`). makes migration kinda a pain

Reply:
I just checked again and confirmed that PR _did not_ change the size, so something else is going on here.

<img width="916" alt="image" src="https://user-images.githubusercontent.com/4071474/165358399-1ac44465-4b14-4453-b5d7-3ebdc26c1a01.png">

Reply:
But it did change the viewer i18n module: 
<img width="754" alt="image" src="https://user-images.githubusercontent.com/4071474/165359625-3efec705-7a00-429e-b471-2089f777f42f.png">

in hindsight none of this lodash changes touched a report, except the i18n stuff, but only for the module/optional report code.

Reply:
yeah the dt bundle increased in size, which def. uses lodash. oops :( 

<img width="891" alt="image" src="https://user-images.githubusercontent.com/4071474/165360211-db522590-2e71-4838-a28a-59f3c6761463.png">

Reply:
looking at the imported module for lodash...it's a big mess of IIFE so no way treeshaking would work

but i think we can use modules directly: `lodash/isEqual.js`. I'll try that and maybe open a pr.

Reply:
#13914